src/pudl/extract/excel.py

```diff
@@ -81,7 +81,7 @@ def get_skipfooter(self, year, page):
 
     def get_column_map(self, year, page):
         """Returns the dictionary mapping input columns to pudl columns for given year and page."""
-        return {v: k for k, v in self._column_map[page].T.loc[str(year)].to_dict().items()}
+        return {v: k for k, v in self._column_map[page].T.loc[str(year)].to_dict().items() if v != -1}
```

Comment:
this nice little fix! i bet it took way longer to track the problem down then to fix it.

Reply:
Unfortunately yes! But glad it didn't run into too many problems during the test. Initially it was > -1 but that wasn't compatible with the 860 format.